app/src/main/java/com/example/dragonx/presentation/RocketList/RocketListParser.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.dragonx.presentation.RocketList
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.example.dragonx.NetworkService.NetworkService
+import com.example.dragonx.NetworkService.Rocket
+import com.example.dragonx.NetworkService.Rockets
+import com.example.dragonx.util.RocketModel
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+enum class ApiStatus {LOADING, ERROR, DONE}
+
+class RocketListParser {
```

Comment:
Read about Clean Architecture principles. It will prompt you how classes should be named.